src/apps/schedule/domain/constants.py

```diff
@@ -43,3 +43,8 @@ class DefaultEvent(BaseModel):
     user_id: uuid.UUID | None = None
     activity_id: uuid.UUID | None
     flow_id: uuid.UUID | None
+
+
+class AvailabilityType(str, Enum):
+    AlwaysAvailable = "AlwaysAvailable"
+    ScheduledAccess = "ScheduledAccess"
```

Comment:
ENUM names should be in Uppercase to follow consistency. Also, you can use space in the enum value I guess